lib/zstd.h

```diff
@@ -356,6 +356,8 @@ typedef enum {
                               * Deviating far from default value will likely result in a compression ratio decrease.
                               * Special: value 0 means "automatically determine hashRateLog". */
 
+    ZSTD_c_enableDedicatedDictSearch=170,
```

Comment:
general rule for new advanced parameters :
- push it into experimental section (must be done in 2 parts, the `enum`, and then the `#define`)
- document what it does

When there is no strong expectation to become "stable"  on short term : 
- give it a high ID (> 1000)